Neos.Cache/Classes/Backend/RedisBackend.php

```diff
@@ -35,67 +35,53 @@
  *  - database:        The database index that will be used. By default,
  *                     Redis has 16 databases with index number 0 - 15
  *  - password:        The password needed for redis clients to connect to the server (hostname)
+ *  - batchSize:       Maximum number of parameters per query for batch operations
  *
  * Requirements:
- *  - Redis 2.6.0+ (tested with 2.6.14 and 2.8.5)
- *  - phpredis with Redis 2.6 support, e.g. 2.2.4 (tested with 92782639b0329ff91658a0602a3d816446a3663d from 2014-01-06)
+ *  - Redis 6.0.0+
+ *  - phpredis with Redis 6.0 support
  *
  * Implementation based on ext:rediscache by Christopher Hlubek - networkteam GmbH
  *
  * Each Redis key contains a prefix built from the cache identifier,
  * so one single database can be used for different caches.
  *
  * Cache entry data is stored in a simple key. Tags are stored in Sets.
- * Since Redis < 2.8.0 does not provide a mechanism for iterating over keys,
- * a separate list with all entries is populated
  *
  * @api
  */
 class RedisBackend extends IndependentAbstractBackend implements TaggableBackendInterface, IterableBackendInterface, FreezableBackendInterface, PhpCapableBackendInterface, WithStatusInterface
 {
     use RequireOnceFromValueTrait;
 
-    const MIN_REDIS_VERSION = '2.6.0';
+    public const MIN_REDIS_VERSION = '6.0.0';
 
     /**
      * @var \Redis
      */
     protected $redis;
 
-    /**
-     * @var integer Cursor used for iterating over cache entries
-     */
-    protected $entryCursor = 0;
+    protected ?bool $frozen = null;
```

Comment:
```suggestion
    protected bool $frozen = false;
```
since the getter declares its return type as bool that would conflict anyway.

Reply:
I assume it was done like that because when it's `null` the getter retrieves the value from Redis and "caches" it this way.